src/Combobox/index.test.js

```diff
@@ -183,17 +183,4 @@ describe("Combobox", () => {
     fireEvent.blur(input);
     expect(screen.getByDisplayValue("New York")).toBeInTheDocument();
   });
-
-  it("should not update input value in uncontrolled manner when `inputValue` is passed", () => {
-    const value = "Initial value unhandled by parent state";
-    render(
-      <Combobox label={LABEL} inputValue={value}>
-        {STATE_ITEMS}
-      </Combobox>
-    );
-    const input = screen.getByPlaceholderText(LABEL);
-    expect(screen.getByDisplayValue(value)).toBeInTheDocument();
-    fireEvent.change(input, { target: { value: "New value" } });
-    expect(screen.getByDisplayValue(value)).toBeInTheDocument();
-  });
```

Comment:
It looks like I have to clean up Combobox a bit now that `value` works as intended.